apps/web/src/app/core/event.service.ts

```diff
@@ -372,7 +372,10 @@ export class EventService {
         msg = humanReadableMsg = this.i18nService.t("enabledSso");
         break;
       case EventType.Organization_DisabledSso:
-        msg = humanReadableMsg = this.i18nService.t("disabledSso");
+        msg = humanReadableMsg =
+          this.i18nService.translationLocale === "en"
+            ? this.i18nService.t("ssoTurnedOff")
+            : this.i18nService.t("disabledSso");
```

Comment:
```suggestion
        msg = humanReadableMsg = this.i18nService.t("ssoTurnedOff");
```
@BTreston meant to use a different key, because just updating the message would only reflect the change in meaning in the source language (en), but would not delete existing translations.

So we need to introduce a new key, which you did with `ssoTurnedOff` and that will need to be translated via Crowdin.

Reply:
Ah I see, that makes sense, thank you!